entities/order.rb

```diff
@@ -0,0 +1,21 @@
+require "/Users/kirillgavris/Desktop/Library/extentions/validations.rb"
+require "/Users/kirillgavris/Desktop/Library/entities/book.rb"
+require "/Users/kirillgavris/Desktop/Library/entities/reader.rb"
+
+class Order
+    include Validations
+
+    attr_accessor :book, :reader, :date
+
+    def initialize(book, reader, date = Time.now)
```

Comment:
This does not validate the date. You just set a default value for this argument.